### PR TITLE
feat: upgrade arb crv strats to new gauge

### PR DESCRIPTION
**Pull request by btcookies**
Handles #452 

Setting gauge and simulating harvest for tricrypto and renbtc crv strategies on arb:
Screen Shot 2022-05-17 at 10 09 13 AM

### CODE REVIEW

**Review by shuklaayush — DISMISSED** [review]
LGTM. Neat script!

File: helpers/addresses.py [hunk 1]
```diff
@@ -627,7 +627,9 @@
     },
     "logic": {
         "native.renCrv": "0x2eE1E845b601371608b6bA4a4665180dE3b14C85",
+        "native.renCrv_v2": "0x021ea7548Ee9B40d57f47706A605043B05C6c92C",
         "native.tricrypto": "0xd64E77C7C6A1dcC7e302F8fe31A22745e223c39c",
+        "native.tricrypto_v2": "0x4Da27cD2AE34a9E1776Ed01747A071C17Fa0b2Cf"
```
[the remark this answers is not in the document]
Comment (btcookies) [reply helpers/addresses.py:632]: i kind of like idea of keeping record of what we're doing, let me reformat though i have other idea

File: scripts/issue/452/upgrade_arb_crv_gauges.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import ADDRESSES_ARBITRUM
+from helpers.addresses import registry
+
+console = Console()
+
+DEV_MULTI = registry.arbitrum.badger_wallets.dev_multisig
+PROXY_ADMIN = registry.arbitrum.proxyAdminDev
+CRV = registry.arbitrum.treasury_tokens.CRV
```
[the remark this answers is not in the document]
Comment (btcookies) [reply scripts/issue/452/upgrade_arb_crv_gauges.py]: removed

**Review by sajanrajdev — APPROVED** [review]
Confirmed new logics to include the changes introduced for this issue and to have been deployed by a deployer seeded by `ops_deployer2`.

Script upgrades the logics, checks for storage to be unmodified, sets new gauge and sets gaugeFactory contract. This will fix the harvest issues.

Additionally includes a simulation to ensure that harvests will work after executing the above.

LGTM

**Review by shuklaayush — APPROVED** [review]

**Pull request merged**